### PR TITLE
feat(automate): add script projects such as statistics and document a…

### PR DESCRIPTION
**Pull request by zzcr**
…utomatic error correction

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: internals/automate/src/count.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import path from 'node:path'
+import fs from 'fs-extra'
+
+const demo = path.resolve('../../examples/sites/demos/pc/app')
```
Comment (kevinmoch) [inline internals/automate/src/count.ts:4]: 在构建文件路径时，最好使用 path.join 或 path.resolve 来确保路径的正确性和可移植性。这可以避免在不同操作系统上的路径问题

File: internals/automate/src/count.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+let apiSize = 0
+let apiLength = 0
+let compNumber = 0
+
+// 递归遍历所有的组件，然后依次修改文件内容和修改文件格式
+const findAllpage = (packagesPath) => {
```
Comment (kevinmoch) [inline internals/automate/src/count.ts:11]: 没有对文件读取和文件统计过程中可能发生的错误进行 try catch 处理

File: internals/automate/src/count.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  if (fs.statSync(packagesPath).isDirectory()) {
+    fs.readdirSync(packagesPath).forEach((childPatch) => {
+      findAllpage(path.join(packagesPath, childPatch))
+    })
+  } else {
+    if (packagesPath.includes('webdoc') && packagesPath.endsWith('.js')) {
```
Comment (kevinmoch) [inline internals/automate/src/count.ts:17]: 可以考虑将文件处理逻辑拆分成更小的函数，提高代码的可读性和维护性。例如，将文件读取和统计逻辑拆分成单独的函数
```javascript
const readAndProcessFile = (filePath) => {
  const stats = fs.statSync(filePath);
  const string = fs.readFileSync(filePath, 'utf8').replace(/\s/g, '');
  compNumber++;
  apiLength += string.length;
  apiSize += stats.size;
};

const findAllpage = (packagesPath) => {
  try {
    if (fs.statSync(packagesPath).isDirectory()) {
      fs.readdirSync(packagesPath).forEach((childPatch) => {
        findAllpage(path.join(packagesPath, childPatch));
      });
    } else {
      if (packagesPath.includes('webdoc') && packagesPath.endsWith('.js')) {
        readAndProcessFile(packagesPath);
      }
    }
  } catch (error) {
    console.error(`Error processing ${packagesPath}: ${error.message}`);
  }
};
```

File: internals/automate/src/count.ts (hunk 1 above)
Comment (zzcr) [reply internals/automate/src/count.ts:4]: 已修复此问题：https://github.com/opentiny/tiny-vue/pull/1187

File: internals/automate/src/count.ts (hunk 2 above)
Comment (zzcr) [reply internals/automate/src/count.ts:11]: 已修复此问题：https://github.com/opentiny/tiny-vue/pull/1187

File: internals/automate/src/count.ts (hunk 3 above)
Comment (zzcr) [reply internals/automate/src/count.ts:17]: 已修复此问题：https://github.com/opentiny/tiny-vue/pull/1187

**Pull request merged**